New package: PredatH0r.ChanSort version 2025-03-08

DandelionSprout opened the pull request:
Checklist for Pull Requests
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.10 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.10.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/269090)

DandelionSprout committed:
New package: PredatH0r.ChanSort version 2025-03-08

stephengillie commented:
Automatic Validation ended with:
>  Executable C:\Users\Validator\AppData\Local\Microsoft\WinGet\Links\chansort.exe returned exit code: -532462766
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | E0434352 | -532462766 | 3762504530 | EXCEPTION_COMPLUS | Unhandled Exception | 


(Automated response - build 1078.)

DandelionSprout committed:
Added what I think could be a dependency.

Based on https://github.com/PredatH0r/ChanSort/blob/master/readme.md#System%20requirements

DandelionSprout committed:
Update PredatH0r.ChanSort.installer.yaml

DandelionSprout committed:
Remove the dependency listing or something?

DandelionSprout commented:
From InstallationVerificationResult.json:

`{"OverallResult":"ManualReview","TestPlansResults":[{"TestPlan":"Validation-Executable-Error","Result":"ManualReview","Metadata":"","ResultMetadata":{"FieldsMetadata":[{"Field":"Architecture","Message":"x86"},{"Field":"Scope"},{"Field":"Locale"}]}}],"AnalysisResults":[{"AnalysisType":"ArpEntry","FieldsMetadata":[{"Field":"Architecture","Message":"x86"},{"Field":"Scope"},{"Field":"Locale"}],"Diagnostics":{"diagnostics":{"changedEntryCount":"1","correlationMeasures":[],"intersectionCount":1,"matchedEntryCount":1,"reason":"normalization match and new/changed"},"installerHash":"0bd4bc4b-9af6-48f7-9316-1ff0fd10f10a","metadata":{"historical":[],"metadata":[{"AppsAndFeaturesEntries":[{"DisplayName":"ChanSort","DisplayVersion":"2025-03-08","InstallerType":"portable","ProductCode":"predath0r.chansort__defaultsource","Publisher":"PredatH0r"}],"installerHash":"0bd4bc4b-9af6-48f7-9316-1ff0fd10f10a","submissionIdentifier":"1cff667f-19cb-4656-aa8b-810192fe27e8"}],"productVersionMax":"2025-03-08","productVersionMin":"2025-03-08"},"status":"Success","submissionData":{"submissionIdentifier":"1cff667f-19cb-4656-aa8b-810192fe27e8"},"version":"1.0"}},{"AnalysisType":"ExeRunInfo","FieldsMetadata":[{"Field":"Architecture","Message":"x86"},{"Field":"Scope"},{"Field":"Locale"}],"Diagnostics":{"C:\\Users\\Validator\\AppData\\Local\\Microsoft\\WinGet\\Links\\chansort.exe":{"ErrorStream":"Unhandled Exception: System.IO.FileNotFoundException: Could not load file or assembly 'DevExpress.Utils.v23.2, Version=23.2.5.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a' or one of its dependencies. The system cannot find the file specified.   at ChanSort.Ui.Program.Main()","OutputStream":"","ExitCode":-532462766,"ExecutionStatusResult":"Fail"}}}],"Metadata":[]}`

The file it claims it fails to load, DevExpress.Utils.v23.2, is in fact in the folder:

<details><summary>Screenshot of the folder:</summary>

![image](https://github.com/user-attachments/assets/e8ecfb8f-efb8-47fa-9d4d-b5b4d4638f42)

</details>

ItzLevvie commented:
@wingetbot run

stephengillie commented:
Automatic Validation ended with:
>  Executable C:\Users\Validator\AppData\Local\Microsoft\WinGet\Links\chansort.exe returned exit code: -532462766
 
 | Hex | Dec | Inverted Dec | Symbol | Description | 
 | --- | --- | --- | --- | --- | 
 | E0434352 | -532462766 | 3762504530 | EXCEPTION_COMPLUS | Unhandled Exception | 


(Automated response - build 1078.)

ItzLevvie commented:
[Policy] Needs-Manual-Merge
